packages/dev/gui/src/3D/controls/touchHolographicV3Button.ts

```diff
@@ -0,0 +1,1028 @@
+import type { AbstractMesh } from "core/Meshes/abstractMesh";
+import type { Control3D } from "./control3D";
+import type { Mesh } from "core/Meshes/mesh";
+import type { Nullable } from "core/types";
+import type { Observer } from "core/Misc/observable";
+import type { Scene } from "core/scene";
+import type { Vector3WithInfo } from "../vector3WithInfo";
+
+import { AdvancedDynamicTexture } from "../../2D/advancedDynamicTexture";
+import { Animation } from "core/Animations/animation";
+import { AnimationGroup } from "core/Animations/animationGroup";
+import { Color3, Color4 } from "core/Maths/math.color";
+import { Control } from "../../2D/controls/control";
+import { CreatePlane } from "core/Meshes/Builders/planeBuilder";
+import { CreateBox } from "core/Meshes/Builders/boxBuilder";
+import { DomManagement } from "core/Misc/domManagement";
+import { FadeInOutBehavior } from "core/Behaviors/Meshes/fadeInOutBehavior";
+import { Grid } from "../../2D/controls/grid";
+import { Image } from "../../2D/controls/image";
+import { MRDLBackglowMaterial } from "../materials/mrdl/mrdlBackglowMaterial";
+import { MRDLBackplateMaterial } from "../materials/mrdl/mrdlBackplateMaterial";
+import { MRDLFrontplateMaterial } from "../materials/mrdl/mrdlFrontplateMaterial";
+import { MRDLInnerquadMaterial } from "../materials/mrdl/mrdlInnerquadMaterial";
+import { Rectangle } from "../../2D/controls/rectangle";
+import { SceneLoader } from "core/Loading/sceneLoader";
+import { StackPanel } from "../../2D/controls/stackPanel";
+import { StandardMaterial } from "core/Materials/standardMaterial";
+import { TextBlock } from "../../2D/controls/textBlock";
+import { TouchButton3D } from "./touchButton3D";
+import { TransformNode } from "core/Meshes/transformNode";
+import { Vector3 } from "core/Maths/math.vector";
+
+/**
+ * Class used to create the mrtkv3 button
+ */
+export class TouchHolographicV3Button extends TouchButton3D {
+    /**
+     * Base Url for the frontplate model.
+     */
+    public static FRONTPLATE_MODEL_BASE_URL = "https://raw.githubusercontent.com/chinezenwosu-ms/TempAssets/main/GLB/";

```

Comment:
We can upload the assets to our assets library (https://github.com/BabylonJS/Assets). It will be available on our CDN.